api/annotations.py

```diff
@@ -166,9 +166,8 @@ def parse(cls, _db, data, patron):
         else:
             content = None
 
-        annotation, is_new = get_one_or_create(
-            _db, Annotation,
-            patron=patron,
+        annotation, is_new = Annotations.get_one_or_create(
+            _db, patron=patron,
```

Comment:
This makes sure that a poorly written client can't accidentally sync annotations when the patron has opted out.